lib/metrician/middleware/request_timing.rb

```diff
@@ -51,7 +51,7 @@ def call(env)
             if Middleware.error?
               # We to_i the status because in some circumstances it is a
               # Fixnum and some it is a string. Because why not.
-              increment(:error, current_route) if status.to_i >= 500
+              increment(:error, current_route) if $! || status.to_i >= 500
```

Comment:
We've had `$ERROR_INFO` since like ruby 1998. You should use that, as it is a more scrutable thing.

https://github.com/ruby/ruby/blob/trunk/lib/English.rb#L51